test/runtests.jl

```diff
@@ -58,4 +58,9 @@ using Test
         @info("Testing inline code")
         include("inline_test.jl")
     end
+
+    # @testset "Notebooks" begin
+    #     @info("Testing Jupyter options")
+    #     include("notebooks.jl")
+    # end
 end
```

Comment:
It would be best to uncomment this so that it can run, it looks like Conda and IJulia installed properly

Reply:
Ah, yeah, good point.